modules/covid_vaccine/app/models/covid_vaccine/v0/expanded_registration_submission.rb

```diff
@@ -47,19 +47,13 @@ class ExpandedRegistrationSubmission < ApplicationRecord
       after_initialize do |reg|
         reg.form_data&.symbolize_keys!
       end
-
-      attr_encrypted :form_data, key: Settings.db_encryption_key, marshal: true, marshaler: JsonMarshal::Marshaller
-      attr_encrypted :raw_form_data, key: Settings.db_encryption_key, marshal: true, marshaler: JsonMarshal::Marshaller
-      attr_encrypted :eligibility_info, key: Settings.db_encryption_key, marshal: true,
-                                        marshaler: JsonMarshal::Marshaller
-
       serialize :eligibility_info, JsonMarshal::Marshaller
       serialize :form_data, JsonMarshal::Marshaller
       serialize :raw_form_data, JsonMarshal::Marshaller
 
-      encrypts :form_data, migrating: true, **lockbox_options
-      encrypts :eligibility_info, migrating: true, **lockbox_options
-      encrypts :raw_form_data, migrating: true, **lockbox_options
+      encrypts :form_data, **lockbox_options
```

Comment:
Super nitpicky but could these three lines just be on one line?

encrypts :form_data, :eligibility_info, :raw_form_data, **lockbox_options